README.md

```diff
@@ -18,12 +20,53 @@ Some layers include the corresponding OTel language SDK for the Lambda. This all
 * ### [.NET + Collector Lambda Layer](dotnet/README.md)
 * ### [Collector Lambda Layer](collector/README.md)
 
-
 ## FAQ
+
 * **What exporters/recievers/processors are included from the OpenTelemetry Collector?**
     > You can check out [the stripped-down collector's imports](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/collector/lambdacomponents/default.go#L18) in this repository for a full list of currently included components.
 * **Is the Lambda layer provided or do I need to build it and distribute it myself?**
     > This repository does not provide pre-build Lambda layers. They must be built manually and saved in your AWS account. This repo has files to facilitate doing that. More information is provided in [the Collector folder's README](collector/README.md).
 
 ## Design Proposal
+
 To get a better understanding of the proposed design for the OpenTelemetry Lamda extension, you can see the [Design Proposal here.](docs/design_proposal.md)
+
+## Features
+
+The following is a list of features provided by the OpenTelemetry layers.
+
+### Custom Context Propagation
```

Comment:
```suggestion
### Custom Context Propagation Carrier Extraction
```

All SDKs support custom context propagation.  What I think we're discussing here is the functionality to provide a hook to extract the carrier (or context, but we should be changing those implementations) from the incoming event.